### ISSUE TITLE
Image Size is not correctly identified

### ISSUE THREAD

**Issue by timmjd** [zynga/jasy#39]
The size of images exported from Photoshop 6.0 with "save as" will be identified with an incorrect size.
Save for Web and Devices works.

Correct size: 760x538
Jasy identifies: 160x113

https://www.dropbox.com/s/x8d2d2y5gr2gfyq/broken.jpg

**Comment by swernerx** [comment]
Fixed with that commit: 9a69840de29f76208f801ec3485b2b781c31aa29

**Comment by swernerx** [comment]
Test Code (the fixed image was re-saved via Mac OS X Preview App):

```
from jasy.asset.ImageInfo import ImgInfo

print(ImgInfo("/Users/swerner/Downloads/broken.jpg").getSize())
print(ImgInfo("/Users/swerner/Downloads/broken_fixed.jpg").getSize()
```

Result:
(760, 538)
(760, 538)
